scripts/ci/wasm_unpack.js

```diff
@@ -0,0 +1,235 @@
+(function(source, uncompressedSize) {
```

Comment:
At a glance the JS here looks fine but I haven't reviewed it carefully. Honestly, maintaining something like this without any tests makes me a bit uncomfortable.

Reply:
Hm... it only ever needs to decode one and the same input... I can hash the wasm file, encode it and then decode it and check the hash again in the ``pack_soljson.sh`` script, then we can be sure that it works for us :-)...

Reply:
Actually we can even load the resulting ``soljson.js`` itself and just write back the contents of ``Module["wasmBinary"]`` and compare to the wasm file, that would test it even more immediately.

Reply:
Cool. Having something like that would be a great way to ensure it's correct.

Reply:
But you're right, it makes sense to try and test it properly at least on the one input it has to act. I'll see if I can simulate all relevant cases, in particular if I can simulate ``Uint8Array.copyWithin`` being available and being unavailable.
If that works within ``node`` then, I'd be pretty content, though - I wouldn't expect JS code to randomly do something else on different platforms or anywhere ever apart from those explicit polyfill-case-distinctions...